packages/trader/src/Stores/Modules/Contract/contract-replay-store.js

```diff
@@ -128,6 +129,11 @@ export default class ContractReplayStore extends BaseStore {
         }
         this.prev_indicative = new_indicative;
 
+        const is_forward_starting =
+            !!this.contract_info.is_forward_starting ||
```

Comment:
Wouldn't the `this.contract_info.is_forward_starting` flag alone be enough? 🤔 
Are there any instances where the `purchase_time !== date_start` not equal and `is_forward_starting` is `0`?

Reply:
@zaki-binary Yes, for expired contracts, `is_forward_starting` is `0`.

Reply:
@ali-fs , ok, but this sounds like an API bug worth raising. Forward starting contracts should still have the `is_forward_starting` regardless of it being sold / expired or open.